src/components/totals/Goal/index.js

```diff
@@ -10,10 +10,11 @@ module.exports = React.createClass({
   displayName: "Goal",
   propTypes: {
     goal: React.PropTypes.string,
-    renderIcon: React.PropTypes.bool,
+    renderIcon: React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.bool]),
     backgroundColor: React.PropTypes.string,
     textColor: React.PropTypes.string,
     format: React.PropTypes.string,
+    centsToDollars: React.PropTypes.bool,
```

Comment:
I don't see this property used anywhere else, is this meant to be `handleCents`  as showin in default props?